### PR TITLE
migmap does create blast databases relative to the conda directory

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

FOR REVIEWER:
* [x] .shed.yml file ok
    - [x] Toolshed user `iuc` has access to associated toolshed repo(s)
* [x] Indentation is correct (4 spaces)
* [x] Tool version/build ok
* [x] `<command/>`
  - [x] Text parameters, input and output files `'single quoted'`
  - [x] Use of `<![CDATA[ ... ]]>` tags
  - [x] Parameters of type `text` or having `optional="true"` attribute are checked with `if str($param)` before being used
* [x] Data parameters have a `format` attribute containing datatypes recognised by Galaxy
* [x] Tests
  - [x] Parameters are reasonably covered
  - [x] Test files are appropriate
* [x] Help
  - [x] Valid restructuredText and uses `<![CDATA[ ... ]]>` tags
* [x] Complies with other best practice in [Best Practices Doc](http://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html)

### CODE REVIEW

**Comment by bgruening** [comment]
ping @bwlang 
Is the second report output really needed as default?

File: tools/migmap/migmap.xml [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+<tool id="migmap" name="MiGMAP" version="1.0.2.2">
     <description>mapper for full-length T- and B-cell repertoire sequencing
     </description>
     <requirements>
         <requirement type="package" version="1.0.2">migmap</requirement>
     </requirements>
     <command detect_errors='exit_code'><![CDATA[
 export IGBLAST_PATH=\$(dirname \$(which igblastn)) &&
+mkdir temp_db &&
+cp -r "\${IGBLAST_PATH}/../share/igblast/" ./temp_db &&
```
Comment (bwlang) [inline tools/migmap/migmap.xml]: why cp here instead of ln -s ?

**Comment by bgruening** [comment]
@bwlang good call, updated it.

**Comment by bwlang** [comment]
I didn't answer your original question... i'm not sure we need the txt output format either.
I think it's probably not needed - and i'm not even sure what those numbers at the end are

**Comment by bgruening** [comment]
@bwlang thanks for your review.

**Comment by mvdbeek** [comment]
Thanks @bgruening @bwlang !

**Pull request merged**